server/routes/postcode.js

```diff
@@ -13,6 +13,7 @@ module.exports = [
       request.yar.set('postcode', null)
       const error = request.query.error
       request.yar.set('previousPage', request.path)
+      const backLinkUri = 'https://www.gov.uk/check-long-term-flood-risk'
```

Comment:
This shouldn't be hard coded. We already have a config value for this `config.floodRiskUrl`, that should be used instead.